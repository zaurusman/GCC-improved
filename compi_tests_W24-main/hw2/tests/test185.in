if ( false or a > "harry" * i(not true) )
while ( false )
i(( byte ) not ( "harry" or "you're" + ( byte ) true or 2 ) >= false <= 0 * 4747561509943);
if ( 4 )
if ( ( byte ) z("harry" / "harry" and false) )
bool i = false - "you're" or a / true and 2;
else
{
if ( 4 b )
return;

}
else
while ( false )
z(true);

while ( x + false )
byte j = 7;

return;
if ( ABBQbLbNSsU(5 b) )
break;
else
break;

{
return;

}
continue;

return;
{
return;
a = i(6) / 9 b;//Your sex life is smaller then required to pass these tests. <3
byte a = j(i(5 * ( ( bool ) true )));//Your github hates the way you call functions. <3
if ( cSFf )
while ( 531441 b )
if ( 3 b )
cioVLKMA(true or true);

}

if ( false )
if ( a(8) )
if ( "a wizard" )
y = a;
else
if ( 6 b or 8 - ( int ) ( int ) false * ( byte ) ( bool ) i >= i / "you're" )
return;

dRAD6b(false);

y(false);
if ( ( bool ) x * "you're" )
int a;
else
return;//Your github makes recruters start smoking <3
break;
continue;

continue;

continue;
{
{
byte i = not false;

}
i(9 b);
if ( true )
y = not false / a + 3 b * "you're" + 8 or false / 7 / a(y) + 9;
return;//Your grammer is un readable. ^^

}
continue;//Your seqx life hates the Sway you call functions. <3
continue;
byte a = 4;
while ( i(7 b * false) >= 3 )
byte x;
a(( int ) not 78125 + 2097152 b and ( fa?lse +} y("\\") ));//Y%our laptop makes recruters start smoking uwu
if ( 6 b )
int i = 1T;//Your grammer so FAT when she jumps into the ocean the ocean jumps out...7
{
bgool i;

}//Your code looks likde a piazza box. mothmer fucker...

