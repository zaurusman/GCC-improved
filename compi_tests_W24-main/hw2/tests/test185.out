27: Exp -> FALSE
21: Exp -> ID
25: Exp -> STRING
26: Exp -> TRUE
28: Exp -> NOT Exp
15: Call -> ID LPAREN Exp RPAREN
22: Exp -> Call
20: Exp -> Exp BINOP Exp
31: Exp -> Exp RELOP Exp
30: Exp -> Exp OR Exp
27: Exp -> FALSE
17: Type -> BYTE
25: Exp -> STRING
25: Exp -> STRING
17: Type -> BYTE
26: Exp -> TRUE
32: Exp -> LPAREN Type RPAREN Exp
20: Exp -> Exp BINOP Exp
30: Exp -> Exp OR Exp
23: Exp -> NUM
30: Exp -> Exp OR Exp
19: Exp -> LPAREN Exp RPAREN
28: Exp -> NOT Exp
32: Exp -> LPAREN Type RPAREN Exp
27: Exp -> FALSE
31: Exp -> Exp RELOP Exp
23: Exp -> NUM
23: Exp -> NUM
20: Exp -> Exp BINOP Exp
31: Exp -> Exp RELOP Exp
15: Call -> ID LPAREN Exp RPAREN
8: Statement -> Call SC
12: Statement -> WHILE LPAREN Exp RPAREN Statement
10: Statement -> IF LPAREN Exp RPAREN Statement
2: Statements -> Statement
23: Exp -> NUM
17: Type -> BYTE
25: Exp -> STRING
25: Exp -> STRING
20: Exp -> Exp BINOP Exp
27: Exp -> FALSE
29: Exp -> Exp AND Exp
15: Call -> ID LPAREN Exp RPAREN
22: Exp -> Call
32: Exp -> LPAREN Type RPAREN Exp
18: Type -> BOOL
27: Exp -> FALSE
25: Exp -> STRING
20: Exp -> Exp BINOP Exp
21: Exp -> ID
26: Exp -> TRUE
20: Exp -> Exp BINOP Exp
23: Exp -> NUM
29: Exp -> Exp AND Exp
30: Exp -> Exp OR Exp
6: Statement -> Type ID ASSIGN Exp SC
24: Exp -> NUM B
9: Statement -> RETURN SC
10: Statement -> IF LPAREN Exp RPAREN Statement
2: Statements -> Statement
4: Statement -> LBRACE Statements RBRACE
11: Statement -> IF LPAREN Exp RPAREN Statement ELSE Statement
27: Exp -> FALSE
26: Exp -> TRUE
15: Call -> ID LPAREN Exp RPAREN
8: Statement -> Call SC
12: Statement -> WHILE LPAREN Exp RPAREN Statement
11: Statement -> IF LPAREN Exp RPAREN Statement ELSE Statement
3: Statements -> Statements Statement
21: Exp -> ID
27: Exp -> FALSE
20: Exp -> Exp BINOP Exp
17: Type -> BYTE
23: Exp -> NUM
6: Statement -> Type ID ASSIGN Exp SC
12: Statement -> WHILE LPAREN Exp RPAREN Statement
3: Statements -> Statements Statement
9: Statement -> RETURN SC
3: Statements -> Statements Statement
24: Exp -> NUM B
15: Call -> ID LPAREN Exp RPAREN
22: Exp -> Call
13: Statement -> BREAK SC
13: Statement -> BREAK SC
11: Statement -> IF LPAREN Exp RPAREN Statement ELSE Statement
3: Statements -> Statements Statement
9: Statement -> RETURN SC
2: Statements -> Statement
4: Statement -> LBRACE Statements RBRACE
3: Statements -> Statements Statement
14: Statement -> CONTINUE SC
3: Statements -> Statements Statement
9: Statement -> RETURN SC
3: Statements -> Statements Statement
9: Statement -> RETURN SC
2: Statements -> Statement
23: Exp -> NUM
15: Call -> ID LPAREN Exp RPAREN
22: Exp -> Call
24: Exp -> NUM B
20: Exp -> Exp BINOP Exp
7: Statement -> ID ASSIGN Exp SC
3: Statements -> Statements Statement
17: Type -> BYTE
23: Exp -> NUM
18: Type -> BOOL
26: Exp -> TRUE
32: Exp -> LPAREN Type RPAREN Exp
19: Exp -> LPAREN Exp RPAREN
20: Exp -> Exp BINOP Exp
15: Call -> ID LPAREN Exp RPAREN
22: Exp -> Call
15: Call -> ID LPAREN Exp RPAREN
22: Exp -> Call
6: Statement -> Type ID ASSIGN Exp SC
3: Statements -> Statements Statement
21: Exp -> ID
24: Exp -> NUM B
24: Exp -> NUM B
26: Exp -> TRUE
26: Exp -> TRUE
30: Exp -> Exp OR Exp
15: Call -> ID LPAREN Exp RPAREN
8: Statement -> Call SC
10: Statement -> IF LPAREN Exp RPAREN Statement
12: Statement -> WHILE LPAREN Exp RPAREN Statement
10: Statement -> IF LPAREN Exp RPAREN Statement
3: Statements -> Statements Statement
4: Statement -> LBRACE Statements RBRACE
3: Statements -> Statements Statement
27: Exp -> FALSE
23: Exp -> NUM
15: Call -> ID LPAREN Exp RPAREN
22: Exp -> Call
25: Exp -> STRING
21: Exp -> ID
7: Statement -> ID ASSIGN Exp SC
24: Exp -> NUM B
23: Exp -> NUM
16: Type -> INT
16: Type -> INT
27: Exp -> FALSE
32: Exp -> LPAREN Type RPAREN Exp
32: Exp -> LPAREN Type RPAREN Exp
17: Type -> BYTE
18: Type -> BOOL
21: Exp -> ID
32: Exp -> LPAREN Type RPAREN Exp
32: Exp -> LPAREN Type RPAREN Exp
20: Exp -> Exp BINOP Exp
20: Exp -> Exp BINOP Exp
21: Exp -> ID
25: Exp -> STRING
20: Exp -> Exp BINOP Exp
31: Exp -> Exp RELOP Exp
30: Exp -> Exp OR Exp
9: Statement -> RETURN SC
10: Statement -> IF LPAREN Exp RPAREN Statement
11: Statement -> IF LPAREN Exp RPAREN Statement ELSE Statement
10: Statement -> IF LPAREN Exp RPAREN Statement
10: Statement -> IF LPAREN Exp RPAREN Statement
3: Statements -> Statements Statement
27: Exp -> FALSE
15: Call -> ID LPAREN Exp RPAREN
8: Statement -> Call SC
3: Statements -> Statements Statement
27: Exp -> FALSE
15: Call -> ID LPAREN Exp RPAREN
8: Statement -> Call SC
3: Statements -> Statements Statement
18: Type -> BOOL
21: Exp -> ID
32: Exp -> LPAREN Type RPAREN Exp
25: Exp -> STRING
20: Exp -> Exp BINOP Exp
16: Type -> INT
5: Statement -> Type ID SC
9: Statement -> RETURN SC
11: Statement -> IF LPAREN Exp RPAREN Statement ELSE Statement
3: Statements -> Statements Statement
13: Statement -> BREAK SC
3: Statements -> Statements Statement
14: Statement -> CONTINUE SC
3: Statements -> Statements Statement
14: Statement -> CONTINUE SC
3: Statements -> Statements Statement
14: Statement -> CONTINUE SC
3: Statements -> Statements Statement
17: Type -> BYTE
27: Exp -> FALSE
28: Exp -> NOT Exp
6: Statement -> Type ID ASSIGN Exp SC
2: Statements -> Statement
4: Statement -> LBRACE Statements RBRACE
2: Statements -> Statement
24: Exp -> NUM B
15: Call -> ID LPAREN Exp RPAREN
8: Statement -> Call SC
3: Statements -> Statements Statement
26: Exp -> TRUE
27: Exp -> FALSE
28: Exp -> NOT Exp
21: Exp -> ID
20: Exp -> Exp BINOP Exp
24: Exp -> NUM B
25: Exp -> STRING
20: Exp -> Exp BINOP Exp
20: Exp -> Exp BINOP Exp
23: Exp -> NUM
20: Exp -> Exp BINOP Exp
27: Exp -> FALSE
23: Exp -> NUM
20: Exp -> Exp BINOP Exp
21: Exp -> ID
15: Call -> ID LPAREN Exp RPAREN
22: Exp -> Call
20: Exp -> Exp BINOP Exp
23: Exp -> NUM
20: Exp -> Exp BINOP Exp
30: Exp -> Exp OR Exp
7: Statement -> ID ASSIGN Exp SC
10: Statement -> IF LPAREN Exp RPAREN Statement
3: Statements -> Statements Statement
9: Statement -> RETURN SC
3: Statements -> Statements Statement
4: Statement -> LBRACE Statements RBRACE
3: Statements -> Statements Statement
14: Statement -> CONTINUE SC
3: Statements -> Statements Statement
14: Statement -> CONTINUE SC
3: Statements -> Statements Statement
17: Type -> BYTE
23: Exp -> NUM
6: Statement -> Type ID ASSIGN Exp SC
3: Statements -> Statements Statement
24: Exp -> NUM B
27: Exp -> FALSE
20: Exp -> Exp BINOP Exp
15: Call -> ID LPAREN Exp RPAREN
22: Exp -> Call
23: Exp -> NUM
31: Exp -> Exp RELOP Exp
17: Type -> BYTE
5: Statement -> Type ID SC
12: Statement -> WHILE LPAREN Exp RPAREN Statement
3: Statements -> Statements Statement
16: Type -> INT
23: Exp -> NUM
28: Exp -> NOT Exp
32: Exp -> LPAREN Type RPAREN Exp
24: Exp -> NUM B
20: Exp -> Exp BINOP Exp
line 81: lexical error
